### PR TITLE
Route number styling

### PR DESCRIPTION
**Pull request by hbruch**
## Proposed Changes
This PR
  - moves some bar styling declarations in scss to higher levels, so they can more easily overridden 
  - implements some OKC specific styling
     - scales bus and tram icon so they have some margin
     - removes / defines some classes/styles in the OKC svgs, as DT defines colors/fill
     - adds an sass/themes/overrides.sccss file to contain all okc specific scss styles
     - defines okc specific scss styles to invert route number styling: route colors are used for outline and vehicle symbol on a white background

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request

### CODE REVIEW

File: sass/themes/okc/_theme.scss [hunk 1]
```diff
@@ -23,3 +23,28 @@ nav.top-bar {
...
+			.bar-container {
+				div.bar {
+					border-color: currentColor;
+					background-color: white;
+					border: 1px solid;
+				}
+				.route-number {
+					.vehicle-number-container-v {
+						.vehicle-number {
+							color: black;
```
Comment (derhuerst) [inline sass/themes/okc/_theme.scss]: In `sass/themes/default/_theme.scss`, we have `$black: #333`. Shall we use that?

**Comment by derhuerst** [comment]
>   * scales bus and tram icon so they have some margin

I think this is a very subtle but important change. We might accidentally revert it. Is there another way to achieve proper icon sizes & layout, while keeping size used by all other icons?

**Comment by hbruch** [comment]
> > * scales bus and tram icon so they have some margin
> 
> I think this is a very subtle but important change. We might accidentally revert it. Is there another way to achieve proper icon sizes & layout, while keeping size used by all other icons?

I'd rather argument that the former okc symbol was the subtle but important change, as the vehicles in default/HSL's symbols for bus and tram both have a margin, which would be lost without downscaling bus and tram a bit

File: sass/themes/okc/_theme.scss (hunk 1 above)
Comment (hbruch) [reply sass/themes/okc/_theme.scss]: Yes, will switch this, though we might darken it later a bit to improve contrast.

**Review by derhuerst — APPROVED** [review]
The changes look reasonable, and according to my very quick testing, work as intended.

Is it on purpose that the stop departures view has a different rendering of route numbers?

**Comment by hbruch** [comment]
I did not address the stop departures view as this should be rendered differently anyway

**Pull request merged**